core/ui/ViewTemplate/lazy-loading.tid

```diff
@@ -0,0 +1,12 @@
+title: $:/core/ui/ViewTemplate/lazy-loading
+tags: $:/tags/ViewTemplate
+
+\whitespace trim
+\rules except dash
```

Comment:
Why is this needed?

Reply:
Because it break the `--c` CSS variables. Let me remove this.